elixir/apps/domain/lib/domain/mailer/sync_email/sync_error.html.eex

```diff
@@ -0,0 +1,136 @@
+									</div>
+								</a>
+							</div>
+							<table style="width: 100%;" cellpadding="0" cellspacing="0" role="none">
+								<tr>
+									<td class="sm-px-6" style="border-radius: 4px; background-color: #ffffff; padding: 48px; font-size: 16px; color: #4f4f4f; box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05)">
+										<h1 class="sm-leading-8" style="margin: 0 0 24px; font-size: 24px; font-weight: 600; color: #000000">
+											Identity Provider Sync Error!
+										</h1>
+										<p style="margin: 0; line-height: 24px">
+											An Identity Provider setup in Firezone has failed to sync <%= @provider.last_syncs_failed %> times.
+										</p>
+										<div role="separator" style="line-height: 16px">&zwj;</div>
+										<p style="margin: 0; line-height: 24px;">
+											Below is the last sync error message:
+											<div>
+												<%= @provider.last_sync_error %>
+											</div>
+										</p>
+										<div role="separator" style="background-color: #d1d1d1; height: 1px; line-height: 1px; margin: 32px 0">&zwj;</div>
+										<p style="margin: 0; line-height: 24px;">
+											<span style="font-weight: 500; text-decoration-line: underline; text-underline-offset: 2px">Identity Provider Details</span>
+										</p>
+										<table class="rtl-text-right" style="width: 100%; text-align: left; font-size: 14px; color: #6d6d6d" cellpadding="0" cellspacing="0" role="none">
+											<tr style="border-bottom-width: 1px; background-color: #ffffff">
+												<th scope="row" style="white-space: nowrap; padding-left: 24px; padding-right: 24px; font-weight: 500; color: #3d3d3d">Account</th>
+                        <td style="padding: 4px 24px"><%= @account.name %></td>
+											</tr>
+											<tr style="border-bottom-width: 1px; background-color: #ffffff;">
+												<th scope="row" style="white-space: nowrap; padding-left: 24px; padding-right: 24px; font-weight: 500; color: #3d3d3d;">Account ID</th>
+												<td style="padding: 4px 24px;"><%= @account.id %></td>
+											</tr>
+											<tr style="border-bottom-width: 1px; background-color: #ffffff;">
+												<th scope="row" style="white-space: nowrap; padding-left: 24px; padding-right: 24px; font-weight: 500; color: #3d3d3d;">Provider Name</th>
+												<td style="padding: 4px 24px;"><%= @provider.name %></td>
+											</tr>
+											<tr style="border-bottom-width: 1px; background-color: #ffffff;">
+												<th scope="row" style="white-space: nowrap; padding-left: 24px; padding-right: 24px; font-weight: 500; color: #3d3d3d;">Provider ID</th>
+												<td style="padding: 4px 24px;"><%= @provider.id %></td>
+											</tr>
+										</table>
+										<p></p>
+										<div role="separator" style="background-color: #d1d1d1; height: 1px; line-height: 1px; margin: 32px 0;">&zwj;</div>
+										<p style="margin: 0;">
```

Comment:
Consistent line-height with the text in paragraph 1

```suggestion
										<p style="margin: 0; line-height: 24px;">
```

Reply:
Updated